internal/impl/sentry/processor_capture.go

```diff
@@ -51,6 +51,13 @@ func newCaptureProcessorConfig() *service.ConfigSpec {
 				Example(`root = {"order": {"product_id": "P93174", "quantity": 5}}`).
 				Example(`root = deleted()`),
 
+			service.NewBloblangField("extras").
+				Version("TBD").
```

Comment:
You can drop this, we've been not using this anymore and have different plans to keep this updated.

Reply:
Fixed